build/transpile.js

```diff
@@ -119,6 +119,9 @@ class Transpiler {
             [ /\.parseTimeInForce /g, '.parse_time_in_force'],
             [ /\.parseTradingFees /g, '.parse_trading_fees'],
             [ /\.describeData /g, '.describe_data'],
+            [ /\.defaultHas /g, '.default_has'],
+            [ /\.getOwnMethods /g, '.get_own_methods'],
+            [ /\.populateHasTree /g, '.populate_has_tree'],
             [ /\.randNumber /g, '.rand_number'],
```

Comment:
why is this needed?

Reply:
ah, this is leftover from previous approach, removed it atm